interop-testing/src/main/java/io/grpc/testing/integration/TestServiceImpl.java

```diff
@@ -435,11 +437,14 @@ private ByteString generatePayload(ByteString dataBuffer, int offset, int size)
   }
 
   /** Returns interceptors necessary for full service implementation. */
-  public static List<ServerInterceptor> interceptors() {
-    return Arrays.asList(
+  public static List<ServerInterceptor> interceptors(
+      Collection<ServerInterceptor> optionalInterceptors) {
```

Comment:
Why add this new argument? Interceptors can be combined by just calling intercept() multiple times.